### ISSUE TITLE
Deprecation in favor of rules_distroless

### ISSUE THREAD

**Issue by betaboon** [bazel-contrib/rules_debian_packages#53]
# TL;DR
`rules_debian_packages` is being deprecated in favor of [rules_distroless](https://github.com/GoogleContainerTools/rules_distroless).
Please migrate ;)

# Long Story
Hello everyone,

Back when i started `rules_debian_packages` there was no way to achieve what this ruleset offers.
I had a need, I build a solution.

@thesayyn has made great progress on [rules_distroless](https://github.com/GoogleContainerTools/rules_distroless) and `distroless` [started using it](https://github.com/GoogleContainerTools/distroless/pull/1465).

Thus we came to the agreement to deprecated `rules_debian_packages` so that efforts can be centralized on a future-proof implementation.
